addons/hr_evaluation/hr_evaluation.py

```diff
@@ -1,375 +1,465 @@
-                                  string="State", required=True, copy=False),
-        'survey_id': fields.related('phase_id', 'survey_id', string="Appraisal Form", type="many2one", relation="survey.survey"),
-        'deadline': fields.related('request_id', 'deadline', type="datetime", string="Deadline"),
-    }
-    _defaults = {
-        'state': 'draft'
-    }
-
-    def create(self, cr, uid, vals, context=None):
-        phase_obj = self.pool.get('hr_evaluation.plan.phase')
-        survey_id = phase_obj.read(cr, uid, vals.get('phase_id'), fields=['survey_id'], context=context)['survey_id'][0]
-
-        if vals.get('user_id'):
-            user_obj = self.pool.get('res.users')
-            partner_id = user_obj.read(cr, uid, vals.get('user_id'), fields=['partner_id'], context=context)['partner_id'][0]
-        else:
-            partner_id = None
-
-        user_input_obj = self.pool.get('survey.user_input')
-
-        if not vals.get('deadline'):
-            vals['deadline'] = (datetime.now() + timedelta(days=28)).strftime(DF)
-
-        ret = user_input_obj.create(cr, uid, {'survey_id': survey_id,
-                                              'deadline': vals.get('deadline'),
-                                              'type': 'link',
-                                              'partner_id': partner_id}, context=context)
-        vals['request_id'] = ret
-        return super(hr_evaluation_interview, self).create(cr, uid, vals, context=context)
-
-    def name_get(self, cr, uid, ids, context=None):
-        if not ids:
-            return []
-        reads = self.browse(cr, uid, ids, context=context)
-        res = []
-        for record in reads:
-            name = record.survey_id.title
-            res.append((record['id'], name))
+    _description = "Employee Appraisal"
+    _order = 'date_close, interview_deadline'
+
+    EVALUATION_STATE = [
+        ('new', 'To Start'),
+        ('pending', 'Appraisal Sent'),
+        ('done', 'Done')
+    ]
+
+    @api.model
+    def _set_default_template(self):
+        return self.env.ref('hr_evaluation.email_template_appraisal')
+
+    @api.one
+    def _get_total_appraisal(self):
+        return self.env['survey.user_input'].search([
+            ('survey_id', 'in', [self.appraisal_manager_survey_id.id, self.appraisal_colleagues_survey_id.id, self.appraisal_self_survey_id.id, self.appraisal_subordinates_survey_id.id]),
+            ('type', '=', 'link'), ('evaluation_id', '=', self.id)])
+
+    @api.one
+    @api.depends('state', 'interview_deadline')
+    def _get_tot_sent_appraisal(self):
+        for records in self._get_total_appraisal():
+            self.tot_sent_appraisal = len(records)
+
+    @api.one
+    def _get_tot_completed_appraisal(self):
+        for records in self._get_total_appraisal():
+            self.tot_comp_appraisal = len(records.filtered(lambda r: r.state == 'done'))
+
+    meeting_id = fields.Many2one('calendar.event', string='Meeting')
```

Comment:
General remark: try to order your fields
 - fields related to the employee + mail / color / ...
 - fields related to dates
 - surveys